tests/model_forms/tests.py

```diff
@@ -3102,6 +3111,12 @@ class Meta:
         self.assertEqual(len(form.cleaned_data), 1)
         self.assertSequenceEqual(form.cleaned_data["items"], [core, pear])
 
+    def test_model_choice_null_characters(self):
```

Comment:
Sorry, I didn't really think where best to put this test. I think as the other test is on `ModelMultipleChoiceFieldTests`, let's move this to `ModelChoiceFieldTests`

Reply:
Thank you for the suggestion!